packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -177,6 +177,15 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
       setActivePane('none');
     } else {
       setActivePane('chat');
+      // timeout is required to give the window time to render the sendbox so we have something to send focus to.
+      // TODO: look into another way to select this/ adjust composite to select this better.
```

Comment:
Nit: could you word this comment to be more explicit: `Selecting elements in the DOM via attributes is not stable. We should expose an API from ChatComposite to be able to focus on the TextBox`.

Reply:
yup! will update